client/stylesheets/_toCat.scss

```diff
@@ -3285,6 +3281,10 @@ On click the images will be displayed at normal size to complete the effect
     font-size: 0.9rem;
 }
 
+.profileUserRating ul li span a {
```

Comment:
Merge rule `.profileUserRating ul li span a` with rule on line 3271<br>Selector should have depth of applicability no greater than 2, but was 5<br>Selector `profileUserRating` should be written in lowercase with hyphens